projects/xerces-c/project.yaml

```diff
@@ -1,4 +1,5 @@
 homepage: "https://xerces.apache.org/"
+main_repo: 'https://github.com/google/libprotobuf-mutator.git'
```

Comment:
wrong repo

Reply:
Changed to their `svn` repo for now.
Happy to delete if necessary.